Source/WebKit/Shared/RemoteLayerTree/RemoteScrollingCoordinatorTransaction.h

```diff
@@ -60,6 +63,7 @@ class RemoteScrollingCoordinatorTransaction {
     // Data encoded here should be "imperative" (valid just for one transaction). Stateful things should live on scrolling tree nodes.
     // Maybe RequestedScrollData should move here.
     bool m_clearScrollLatching { false };
+    WebCore::FrameIdentifier m_frameID;
```

Comment:
Which frame? the root frame in the subprocess? A more expressive name would help.

Reply:
Changed to root frame.